docs/02_cva6_requirements/cva6_requirements_specification.rst

```diff
@@ -138,11 +138,13 @@ The CVA6 is highly configurable via SystemVerilog parameters.
 It is not practical to fully document and verify all possible combinations of parameters, so a set of "viable IP configurations" has been defined.
 The full list of parameters for this configuration will be detailed in the users’ guide.
 
-Below is the configuration of the first release of the CVA6.
+Below are the configuration of the first releases of the CVA6.
 
 +--------------------+---------+---------+------+-------+---------+---------+---------+---------+
 | Release ID         | Target  | ISA     | XLEN | FPU   | CV-X-IF | MMU     | L1 D$   | L1 I$   |
 +====================+=========+=========+======+=======+=========+=========+=========+=========+
+| CV32A60B           | ASIC    | IMC     |  32  | No    | Yes     | None    | None    | None    |
```

Comment:
Following a Cores TG meeting:
CV32A60B is not accepted as the "embedded" configuration is not an application core.
The following part numbers are proposed:
- CV32E60X if we keep the single issue
- CV32E6?X (e.g. CV32E62X, CV32E65X...) if the configuration includes the dual issue, to reflect the higher performance. (My side note: don't use CV32E64X that would surely bring confusion).

I suggest to replace CV32A60X by CV32E6?X for this document and we'll update when the single-issue/dual-issue configuration is decived.

Reply:
Do we need to call it "Embedded", "CV32A60B", "CV32E62X", "CV32E65X", "CV32E60X" or "CV32E6?X"   What a mess !!

Reply:
I have updated the PR with a proposal.